core/invoker/src/main/resources/application.conf

```diff
@@ -176,6 +176,8 @@ whisk {
   }
 
   invoker {
+    username: "invoker.user"
```

Comment:
Should there be an equivalent default config for the scheduler?

Also don't know off the top of my head but is `.` a special character for curl requests? Wouldn't want to make it harder to curl the invoker with a special character for those that don't care about creds and just using the default

Reply:
scheduler already has such config https://github.com/apache/openwhisk/blob/master/core/scheduler/src/main/resources/application.conf#L62-L63

and `.` is not special for curl:

```
[raja@localhost ansible]$ curl -u invoker.user:invoker.pass https://localhost:12001/isEnabled -k
{"enabled":true}
```